.github/workflows/tests_mergin_media_sync.yaml

```diff
@@ -0,0 +1,87 @@
+name: Tests for Mergin Maps Media Sync
+
+on:
+  push:
+    paths:
+    - "test/**"
+    - "**.py"
+    - ".github/workflows/tests_mergin_media_sync.yaml"
+
+env:
+  TEST_MERGIN_URL: https://test.dev.merginmaps.com
+  TEST_API_USERNAME: test_media_sync
+  TEST_API_PASSWORD: ${{ secrets.TEST_API_PASSWORD }}
+  TEST_API_WORKSPACE: test-media-sync
+  TEST_MINIO_URL: 127.0.0.1:9000
+  TEST_MINIO_ACCESS_KEY: minioaccesskey
+  TEST_MINIO_SECRET_KEY: miniosecretkey
+
+jobs:
+
+  Tests-for-Mergin-Maps-Media-Sync:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Install Spatialite
+        run: sudo apt-get install -y libsqlite3-mod-spatialite
+
+      - name: Install Python dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install mergin-client pytest pytest-cov dynaconf psycopg2 minio
```

Comment:
there's discrepancy between what pip will install here vs what is specified in pipfile - pip will currently install mergin-client 0.9 while pipfile specifies 0.8.3. And to respect pipfile, we will have to first checkout the repo's code... (and it looks like the same problem is in db-sync github workflow, it ignores requirements.txt - that would be good to fix as well before we get burnt by that :slightly_smiling_face: )